src/components/ContentWrapper.jsx

```diff
@@ -40,20 +42,44 @@ const ContentWrapper = (props) => {
   const [isAuth, setIsAuth] = useState(false);
   const [collapsed, setCollapsed] = useState(false);
 
-  const { experimentId, experimentData, children } = props;
+  const { routeExperimentId, experimentData, children } = props;
   const router = useRouter();
   const route = router?.route || '';
   const navigateTo = useAppRouter();
 
-  const experiment = useSelector((state) => state?.experiments[experimentId]);
+  const currentExperimentIdRef = useRef(routeExperimentId);
+  const activeProjectUuid = useSelector((state) => state?.projects?.meta?.activeProjectUuid);
+  const activeProject = useSelector((state) => state.projects[activeProjectUuid]);
+  const samples = useSelector((state) => state.samples);
+
+  const activeProjectExperimentID = useSelector((state) => (
+    state?.projects[activeProjectUuid]?.experiments[0]));
+  // if the current selected project experiment is processed
+  // and we are in data-management, we don't have experimentId
+  const currentExperimentId = currentExperimentIdRef.current;
```

Comment:
Does moving this code below the `useEffect` affect the code? can you move below the block? I think it's clearer if it's placed there, because then you know what the value of `currentExperimentIdRef`resolves to.